### DISCUSSION TITLE
Change from Firefox to Chrome

### DISCUSSION THREAD

**Discussion by FapGermany** [aclap-dev/video-downloadhelper#804] [License & Purchase requests]
### What's your issue?

My favorite browser has changed over the last few months. I'm now reminded that I don't have a premium version when I download a lot. Then I have to open Firefox again. Is it possible to use the paid add-on in my new favorite browser?

### The 10 first digits (only 10) of your license number or email address

381b9bb527

### I confirm that I did not not paste my full license number

- [x] I confirm

**Comment by jcfrog** [comment]
Hi,
Sorry, you can only use your Firefox premium in Firefox: https://github.com/aclap-dev/video-downloadhelper/wiki#go-premium

**Comment by FapGermany** [comment]
I am aware of that. That is why I am asking whether the Firefox version can be deactivated and a Google Chrome version activated. Because: what do I want with two licenses, one of which I don't use?
